config/webpack/serve.webpack.config.js

```diff
@@ -16,6 +16,32 @@ const hostUtils = require('../../utils/host-utils');
 const moduleLoader = skyPagesConfigUtil.outPath('loader', 'sky-pages-module');
 
 /**
+ * Returns the querystring base for parameters allowed to be passed through.
+ * @name getQueryStringFromArgv
+ * @param {Object} argv
+ * @returns {string}
+ */
+function getQueryStringFromArgv(argv) {
+  const allowed = [
+    'envid',
+    'svcid'
+  ];
+
+  let found = [];
+  allowed.forEach(param => {
+    if (argv[param]) {
+      found.push(`${param}=${argv[param]}`);
```

Comment:
Any time you build a URL from user input you should pass it through the `encodeURIComponent()` method.  A good test for this would be to specify one of the values with an ampersand or question mark and make sure it gets escaped.